contracts/v0.2/src/SubmissionProxy.sol

```diff
@@ -391,6 +396,58 @@ contract SubmissionProxy is Ownable {
         }
     }
 
+    function submitStrict(
+        bytes32[] calldata _feedHashes,
+        int256[] calldata _answers,
+        uint256[] calldata _timestamps,
+        bytes[] calldata _proofs
+    ) external {
+        if (
+            _feedHashes.length != _answers.length || _answers.length != _proofs.length
+                || _proofs.length != _timestamps.length || _feedHashes.length > maxSubmission
+        ) {
+            revert InvalidSubmissionLength();
+        }
+
+        uint256 feedsLength_ = _feedHashes.length;
+        for (uint256 i = 0; i < feedsLength_; i++) {
+            submitSingle(_feedHashes[i], _answers[i], _timestamps[i], _proofs[i]);
+        }
+    }
+
+    function submitSingle(
+        bytes32 _feedHash,
+        int256 _answer,
+        uint256 _timestamp,
+        bytes calldata _proof
+    ) public {
+        if (_timestamp <= block.timestamp - dataFreshness || lastSubmissionTimes[_feedHash] >= _timestamp) {
```

Comment:
Let's apply `forge fmt`.

Reply:
weird, I called forge fmt already, maybe the settings are different